### PR TITLE
Add hotkey configuration for scroll keys

### PR DESCRIPTION
**Pull request by obrakmann**
This is a resurrected #9148.

No changes except to fix the code style issue.

### CODE REVIEW

File: OpenRA.Mods.Common/Widgets/Logic/SettingsLogic.cs [hunk 1]
```diff
@@ -410,6 +410,11 @@ Action InitInputPanel(Widget panel)
 
 					{ "ToggleStatusBarsKey", "Toggle status bars" },
 					{ "TogglePixelDoubleKey", "Toggle pixel doubling" },
+
+                    { "MapScrollUp", "Map scroll up" },
+                    { "MapScrollDown", "Map scroll down" },
+                    { "MapScrollLeft", "Map scroll left" },
+                    { "MapScrollRight", "Map scroll right" }
```
Comment (Mailaender) [inline OpenRA.Mods.Common/Widgets/Logic/SettingsLogic.cs]: Spaces to tabs please.

**Comment by obrakmann** [comment]
Updated.

**Comment by abcdefg30** [comment]
:+1: / :white_check_mark:

**Comment by Mailaender** [comment]
Works as promised. :white_check_mark: Thanks for taking care.

**Comment by Mailaender** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/6251589b74d1d23d7977a0c04a0e62e01cdc0f69...5f76c0f70882132fe436b18926b5aaff54c61fa6)

**Pull request merged**